members/admin/user_admin.py

```diff
@@ -139,3 +142,24 @@ def get_list_filter(self, request):
                 AdminUserUnionListFilter,
                 AdminUserDepartmentListFilter,
             ]
+
+    def get_fieldsets(self, request, obj=None):
+        fieldsets = super(UserAdmin, self).get_fieldsets(request, obj)
+        if not obj:
+            return fieldsets
```

Comment:
Er der en grund til at sætte `fieldsets` variabel, når den i dette tilfælde kun bliver brugt hvis `obj is None`?
```suggestion
        if not obj:
            return super(UserAdmin, self).get_fieldsets(request, obj)
```